packages/web/src/app/doc-pages/components/components-routing.ts

```diff
@@ -84,6 +84,10 @@ export const routes: Routes = [
     path: Pages.Dropdown,
     loadComponent: () => import('./dropdown-doc/dropdown-doc.component').then((m) => m.DropdownDocComponent),
   },
+  {
+    path: Pages.Illustration,
+    redirectTo: `/brand/${Pages.Illustration}`,
+  },
```

Comment:
If we now add an entry to the contentful sidebar for `/components/illustration`, it will redirect to brand from this

Reply:
Has it been added yet?